tests/RobotFramework/tests/tedge_to_te_converter/convert_tedge_topics_to_te_topics.robot

```diff
@@ -30,12 +30,44 @@ Convert child device event topic
 
 Convert main device alarm topic
     Execute Command    tedge mqtt pub tedge/alarms/minor/test_alarm '{"text":"test alarm"}' -q 2 -r
-    Should Have MQTT Messages    te/device/main///a/test_alarm    message_contains=minor
+    ${messages}=    Should Have MQTT Messages    te/device/main///a/test_alarm    minimum=1    maximum=2
```

Comment:
What's the logic behind using `maximum=2`? I would have expected `maximum=1`, so that we check that we don't send two messages instead of one.

Reply:
That's not really necessary, I removed it.

Reply:
No we should use `maximum=1` as well because otherwise the test will also pass if more than 1 message is published.

Reply:
updated